airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/array.sql

```diff
@@ -44,6 +45,18 @@
     cross join table(flatten({{ array_col }})) as {{ array_col }}
 {%- endmacro %}
 
+{% macro sqlserver__cross_join_unnest(stream_name, array_col) -%}
+{# -- https://docs.microsoft.com/en-us/sql/relational-databases/json/convert-json-data-to-rows-and-columns-with-openjson-sql-server?view=sql-server-ver15#option-1---openjson-with-the-default-output -- #}
+{# -- add some aditional explanation here about types -- #}
```

Comment:
Is this a TODO?

Reply:
`{# -- add some aditional explanation here about types -- #}` this comment should be removed, the link above has all information.

Reply:
Done.